### PR TITLE
nowhere is one word

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* nowhere is one word  
  [paulb777](https://github.com/paulb777)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
```RuntimeError: #<Pod::UserInterface::InspectorReporter:0x00000002d150e0> does not handle inspector_successfully_recieved_report
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/sidekick.rb:91:in `block in validate_delegate': In general - handles inspector_successfully_received_report
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/sidekick.rb:90:in `each'
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/sidekick.rb:90:in `validate_delegate'
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/sidekick.rb:20:in `search'
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/inspector.rb:65:in `search_query'
	/home/travis/build/CocoaPods/CocoaPods/vendor/bundle/ruby/2.0.0/gems/gh_inspector-1.1.0/lib/gh_inspector/inspector.rb:59:in `search_exception'
	/home/travis/build/CocoaPods/CocoaPods/lib/cocoapods/user_interface/error_report.rb:119:in `search_for_exceptions'
	spec/unit/user_interface/error_report_spec.rb:123:in `block (3 levels) in <module:Pod>'
	/home/travis/build/CocoaPods/CocoaPods/spec/spec_helper/pre_flight.rb:32:in `call'
	/home/travis/build/CocoaPods/CocoaPods/spec/spec_helper/pre_flight.rb:32:in `block in <class:Context>'
	spec/unit/user_interface/error_report_spec.rb:114:in `block (2 levels) in <module:Pod>'
	spec/unit/user_interface/error_report_spec.rb:9:in `block in <module:Pod>'
	spec/unit/user_interface/error_report_spec.rb:4:in `<module:Pod>'
	spec/unit/user_interface/error_report_spec.rb:3:in `<top (required)>'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli/exec.rb:75:in `load'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli/exec.rb:75:in `kernel_load'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli/exec.rb:28:in `run'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli.rb:424:in `exec'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli.rb:27:in `dispatch'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/cli.rb:18:in `start'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/gems/2.0.0/gems/bundler-1.16.0/exe/bundle:30:in `block in <top (required)>'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/site_ruby/2.0.0/bundler/friendly_errors.rb:122:in `with_friendly_errors'
	/home/travis/.rvm/rubies/ruby-2.0.0-p647/lib/ruby/gems/2.0.0/gems/bundler-1.16.0/exe/bundle:22:in `<top (required)>'
	/home/travis/.rvm/gems/ruby-2.0.0-p647/bin/bundle:23:in `load'
	/home/travis/.rvm/gems/ruby-2.0.0-p647/bin/bundle:23:in `<main>'
	/home/travis/.rvm/gems/ruby-2.0.0-p647/bin/ruby_executable_hooks:15:in `eval'
	/home/travis/.rvm/gems/ruby-2.0.0-p647/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by orta** [comment]
Ah yeah, I have a PR for that

File: lib/cocoapods/generator/copy_resources_script.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@ def script
 set -o pipefail
 
 if [ -z ${UNLOCALIZED_RESOURCES_FOLDER_PATH+x} ]; then
-    # If UNLOCALIZED_RESOURCES_FOLDER_PATH is not set, then there's no where for us to copy
+    # If UNLOCALIZED_RESOURCES_FOLDER_PATH is not set, then there's nowhere for us to copy
```
Comment (dnkoutso) [inline lib/cocoapods/generator/copy_resources_script.rb]: need to bump integration specs for that :)

**Comment by orta** [comment]
https://github.com/orta/gh_inspector/pull/21

File: lib/cocoapods/generator/copy_resources_script.rb (hunk 1 above)
Comment (paulb777) [reply lib/cocoapods/generator/copy_resources_script.rb]: https://github.com/CocoaPods/cocoapods-integration-specs/pull/137

**Comment by segiddins** [comment]
👍

**Comment by segiddins** [comment]
@paulb777 throw in a `bundle update gh_issue_inspector` and this should be ready to merge

**Comment by paulb777** [comment]
@segiddins Throw it where?  Sorry, I'm a bundler novice.

> $ bundle update gh_issue_inspector
> Could not find gem 'gh_issue_inspector'.

**Comment by orta** [comment]
I think it's: `bundle update gh_inspector`

**Comment by paulb777** [comment]
Thanks @orta! That works if I also delete the repo information in the Gemfile for `clintegracon`

**Comment by paulb777** [comment]
It's not totally clear, but I think the only remaining travis error is about the CHANGELOG which shouldn't be needed for this cleanup PR.

**Comment by dnkoutso** [comment]
Merging after i double checked the only failure was lint around changelog.

**Pull request merged**